kubeconfig-creator/README.md

```diff
@@ -0,0 +1,129 @@
+# Kubeconfig Creator Task
+
+This `Task` do a similar job to the [Cluster](https://github.com/tektoncd/pipeline/blob/master/docs/resources.md#cluster-resource) 
+`PipelineResource` and
+are intended as its replacement. This is part of our plan to [offer replacement
+`tasks` for Pipeline Resources](https://github.com/tektoncd/catalog/issues/95)
+as well as
+[document those replacements](https://github.com/tektoncd/pipeline/issues/1369).
+
+This task creates a [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+file that can be used to configure access to the different clusters.
+A common use case for this task is to deploy your `application/function` on different clusters.
+
+The task will use the [kubeconfigwriter](https://github.com/tektoncd/pipeline/blob/master/cmd/kubeconfigwriter/main.go) 
+image and the provided parameters to create a `kubeconfig` file that can be used by other tasks
+in the pipeline to access the target cluster. The kubeconfig will be placed at 
+`/workspace/<workspace-name>/kubeconfig`.
+
+This task provides users variety of ways to authenticate:
+- Authenticate using tokens.
+- Authenticate using client key and client certificates.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/kubeconfig-creator/kubeconfig-creator.yaml
+```
+
+## Workspace
+
+* **output**: A workspace that stores the generated kubeconfig file, such that it can be used in the other tasks to access the cluster.
+
+
+## Parameters
+
+* **Name**: Name of the `cluster`.
+* **Type**: Type of the resource (_default:_
```

Comment:
I don't think this field is used for anything, either by the script of this Task or by `kubeconfigwriter`.  Is that right?

I wonder if we should remove it.  What do you think?

Reply:
Yes, It can be removed.